learn/getting-started/writing-a-grpc-service-with-ballerina.md

```diff
@@ -87,55 +108,66 @@ $ bal grpc --mode service --input helloworld.proto --output greeter_service/
+- The `helloworld_pb.bal` file is the stub file, which contains classes that the client/service uses to talk to each other and the Ballerina types corresponding to the request and response messages.
+- The `greeter_service.bal` file is the service template file, which contains service(s) with all the remote methods
   defined in the `.proto` file.
 
-#### Update and run the application
+### Updating the service template file
 
-Let’s update the service template file (i.e., `greeter_service.bal`) and add the business logic to the remote method. In this case, you only need to update the `sayHello` method as shown below.
+To add the business logic to the remote method (in this case, you only need to update the `sayHello` method as shown below), follow these steps:
 
-```ballerina
-import ballerina/grpc;
+1. Open the `greeter_service` directory in your text editor. 
 
-listener grpc:Listener grpcListener = new (9090);
+    >**Tip:** If you are using VS Code, in the terminal, navigate to the , and execute the `code .` command.
 
-@grpc:ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_HELLOWORLD, descMap: getDescriptorMapHelloworld()}
-service "Greeter" on grpcListener {
+2. Replace the service template file (i.e., `greeter_service.bal`) with the code below.
 
-   remote function sayHello(HelloRequest value) returns HelloReply|error {
-       return { message: "Hello " + value.name};
-   }
-}
-```
+    ```ballerina
+    import ballerina/grpc;
+
+    listener grpc:Listener grpcListener = new (9090);
+
+    @grpc:ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_HELLOWORLD, descMap: getDescriptorMapHelloworld()}
+    service "Greeter" on grpcListener {
+
+      remote function sayHello(HelloRequest value) returns HelloReply|error {
+          return { message: "Hello " + value.name};
+      }
+    }
+    ```
 
-In this code,
-- The Listener declaration creates a new gRPC listener with port 9090. The listener is the entity that listens to the
-input coming to the port and then dispatches it to the correct service(s).
-- The service declaration creates a service and attaches it to the listener. The service annotation is to create an
-  internal mapping between the service declarations and the PROTO definition. Do not change it.
-- The gRPC service can have one or more remote methods depending on the proto definition. Here, this service has only one
-  method called `sayHello` that has the `HelloRequest` type as the request and `HelloReply` type as the response.
+    In this code:
+    - The listener declaration creates a new gRPC listener with port 9090. The listener is the entity that listens to the
+    input coming to the port and then dispatches it to the correct service(s).
+    - The service declaration creates a service and attaches it to the listener. The service annotation is to create an
+      internal mapping between the service declarations and the PROTO definition. Do not change it.
```

Comment:
Should be `.proto defenition` . Refer [1]

[1] https://developers.google.com/protocol-buffers/docs/proto3

Reply:
Fixed.